govc/cluster/override/change.go

```diff
@@ -62,15 +63,27 @@ func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.das.DasSettings = new(types.ClusterDasVmSettings)
 
 	f.StringVar((*string)(&cmd.das.DasSettings.RestartPriority), "ha-restart-priority", "", "HA restart priority: "+strings.Join(rp, ", "))
+
+	f.Var(flags.NewInt32(&cmd.orc.VmReadiness.PostReadyDelay), "ha-additional-delay", "HA Additional Delay")
+
+	rc := []string{
+		string(types.ClusterVmReadinessReadyConditionPoweredOn),
+		string(types.ClusterVmReadinessReadyConditionGuestHbStatusGreen),
+		string(types.ClusterVmReadinessReadyConditionAppHbStatusGreen),
+		string(types.ClusterVmReadinessReadyConditionUseClusterDefault),
+	}
+	f.StringVar((*string)(&cmd.orc.VmReadiness.ReadyCondition), "ha-ready-condition", "", "HA VM Ready Condition (Start next priority VMs when): "+strings.Join(rc, ", "))
 }
 
 func (cmd *change) Description() string {
 	return `Change cluster VM overrides.
-
+	
 Examples:
-  govc cluster.override.change -cluster cluster_1 -vm vm_1 -ha-restart-priority high
-  govc cluster.override.change -cluster cluster_1 -vm vm_2 -drs-enabled=false
-  govc cluster.override.change -cluster cluster_1 -vm vm_3 -drs-enabled -drs-mode fullyAutomated`
+	govc cluster.override.change -cluster cluster_1 -vm vm_1 -drs-enabled=false
```

Comment:
Can you revert the whitespace changes here (and the indent above)?  The `Examples:` section is 2 space indent in all other commands. Keeps the formatting consistent in the generated USAGE.md

Reply:
done